style.css

```diff
@@ -4,6 +4,72 @@
   box-sizing: border-box;
 }
 
+body {
+  background-color: #eeeded;
+}
+
+.book-section {
+  display: block;
+  margin: 0 auto;
+  width: 80%;
+}
+
+.add-section {
+  display: none;
+  margin: 60px auto;
+  width: 80%;
+}
+
+.contact-section {
+  display: none;
+  position: relative;
+  font-size: 20px;
+}
+
+.header-nav {
+  display: flex;
```

Comment:
- [ ] You have used `display: flex` 4 times, which falls under the category of bad CSS practices because you are repeating yourself. I would suggest you create a class which displays flex and then use it in the places that you need

Reply:
Hello @GzimAsani 
I hope you are having a good day, I really think this is going to be helpful, but I didn't understand it well, do you mean like we should create a variable for display flex and then use it in the code or is there another way to approach that issue?

Reply:
Hi @EidHachem,
Yes, I meant you create a class for example:

`
.flex: {
display: flex
}
`

and use it in the html as a class at places that you need

Reply:
Ah that's right, I didn't thought about it this way 😅 thank you again, I really appreciate it.

Reply:
Np, best of luck